### PR TITLE
percentage of participation tokens is now dynamic

### PR DESCRIPTION
**Pull request by bconfortin**
https://github.com/AugurProject/augur/issues/3873

I believe the formula is correct

### CODE REVIEW

File: packages/augur-ui/src/modules/reporting/common.tsx [hunk 1]
```diff
@@ -842,9 +843,10 @@ export interface ParticipationTokensViewProps {
   openModal: Function;
   disputeWindowFees: FormattedNumber;
   purchasedParticipationTokens: FormattedNumber;
-  disputeWindow: string;
+  disputeWindow: DisputeWindow;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/common.tsx]: Is `disputeWindow` used?

File: packages/augur-ui/src/modules/reporting/common.tsx [hunk 2]
```diff
@@ -33,6 +33,7 @@ import ChevronFlip from 'modules/common/chevron-flip';
 import TooltipStyles from 'modules/common/tooltip.styles.less';
 import Styles from 'modules/reporting/common.styles.less';
 import { Getters, convertDisplayValuetoAttoValue, convertAttoValueToDisplayValue } from '@augurproject/sdk';
+import { DisputeWindow } from '@augurproject/sdk/src/state/getter/Universe';
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/common.tsx]: remove this import

File: packages/augur-ui/src/modules/reporting/containers/participation-tokens-view.ts [hunk 3]
```diff
@@ -2,27 +2,34 @@ import { connect } from 'react-redux';
...
+  const purchasedParticipationTokens = formatAttoRep(purchasedTokens);
+  const totalAmount = participationTokens && participationTokens.totalAmount || 0;
+  const percentageOfTotalFees = formatPercent(purchasedParticipationTokens.value ? createBigNumber(totalAmount).dividedBy(createBigNumber(purchasedTokens)) : 0);
   return {
     disputeWindowFees: formatAttoDai(state.universe && state.universe.disputeWindow ? state.universe.disputeWindow.fees : 0),
-    purchasedParticipationTokens: formatAttoRep(state.universe && state.universe.disputeWindow ? state.universe.disputeWindow.purchased : 0),
+    purchasedParticipationTokens,
     tokensOwned: formatAttoRep(tokenAmount),
     participationTokens,
+    disputeWindow,
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/containers/participation-tokens-view.ts]: remove disputeWindow

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 👌

 This link will always point to this PR's latest changes: https://peek.run/p37hv1x

**Review by nuevoalex — APPROVED** [review]
lgtm

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**